test/load/assets/prometheus/prometheus-adapter-deployment.yaml

```
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: metrics-adapter
+    app.kubernetes.io/name: prometheus-adapter
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 0.10.0
+  name: prometheus-adapter
+  namespace: monitoring
+spec:
+  replicas: 2
```

we don't need two replicas here
```suggestion
  replicas: 1
```